kitchen-tests/cookbooks/installthings/attributes/default.rb

```diff
@@ -0,0 +1,8 @@
+case node['platform_family']
+when "debian"
+  default['installthings']['push_client_url'] =
+    'https://opscode-private-chef.s3.amazonaws.com/ubuntu/12.04/x86_64/opscode-push-jobs-client_1.1.5-1_amd64.deb'
```

Comment:
can we find some smaller debs/rpms to install?  also the network dependency here will break at some point in time, if we can get a tiny rpm + deb checked into the cookbook they could just be installed locally.